packages/functional-tests/lib/targets/stage.ts

```diff
@@ -5,7 +5,7 @@ export class StageTarget extends RemoteTarget {
   static readonly target = 'stage';
   readonly name: TargetName = StageTarget.target;
   readonly contentServerUrl = 'https://accounts.stage.mozaws.net';
-  readonly relierUrl = 'https://stage-123done.herokuapp.com';
+  readonly relierUrl = 'https://fxa-stage-123done.herokuapp.com/';
```

Comment:
Q: Why does this URL have a trailing `/`, but didn't previously (or the `contentServerUrl` doesn't have a trailing slash). Wondering if this could create URLs like `https://localhost:8080//home` with double slashes in some edgey cases.